rest/clients/cxf/owasp-suppressions.xml

```diff
@@ -28,4 +28,12 @@
         <cpe>cpe:/a:springsource:spring_framework</cpe>
         <cpe>cpe:/a:pivotal_software:spring_framework</cpe>
     </suppress>
+    <suppress>
+        <!-- Only present when JDOM 1.x or 2.x jar is in the classpath. JDOM has been excluded. -->
```

Comment:
❓Should we add a note in the comment to remove the `maven-enforcer-plugin` from the pom file when this vulnerability doesn't apply anymore?
```suggestion
        <!--
            Only present when JDOM 1.x or 2.x jar is in the classpath. JDOM has been excluded.
            Remove maven-enforcer-plugin from pom file when this suppression is removed.
        -->
```

Reply:
```xml
<!--
    Only present when JDOM 1.x or 2.x jar is in the classpath. JDOM has been excluded in the
    classpath for this project. When the version of jackson-databind has been updated,
    remove this suppression and the maven-enforcer-plugin in pom.xml.
-->
```